client/src/html/owner_dashboard.html

```diff
@@ -0,0 +1,890 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link  rel="stylesheet" href="../css/lib/bootstrap.min.css" type="text/css">
+    <link  rel="stylesheet" href="../css/common.css" type="text/css">
+
+    <script type="text/javascript" src="../js/lib/jquery-1.11.3.min.js"></script>
+    <script type="text/javascript" src="../js/lib/jquery-ui.min.js"></script>
+    <script type="text/javascript" src="../js/lib/bootstrap.min.js"></script>
+    <script type="text/javascript" src="../js/lib/bootstrap-validator.js"></script>
+    <script type="text/javascript" src="../js/owner.js"></script>
```

Comment:
why do we need js file to be loaded on every page? This is needed only once in the main html file